docker-compose.cypress.yml

```diff
@@ -0,0 +1,79 @@
+version: "3.7"
+services:
+  cypress:
+    image: cypress/included:6.0.0
+    environment:
+      NODE_ENV: test
+    entrypoint: []
+    command: /bin/bash -c 'npx wait-on http://ckan:5000/api/action/status_show && npx cypress install && npx cypress run'
```

Comment:
We define `ckan` as an application in our docker-compose file. This is not used in the github action. I'm not sure what the name of the core application will be, but it will be something else. This is probably why the tests are hanging...

Reply:
Neither `localhost` nor `0.0.0.0` works.  I follow what you're saying, but I haven't found a good way of figuring out that url.  For [inventory-app](https://github.com/GSA/inventory-app/blob/main/docker-compose.test.yml#L12), using `app` works, so I thought that it would be the name of the container, like a normal docker-compose application, but it doesn't work here.

Reply:
So, apparently, the main ckan app was failing to start because it didn't detect the dcat_usmetadata extension, so I mapped the extension so it gets installed as part of the prerun.py script.